modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/CollectDigitalSignaturePortlet.java

```diff
@@ -111,6 +119,25 @@ private String _getTitle(String fileEntryTitle, int count) {
 			new String[] {fileEntryTitle, String.valueOf(count)}, false);
 	}
 
+	private JSONArray _jsonArray(long[] fileEntryIds) throws Exception {
```

Comment:
Should be better: `private JSONArray _getJSONArray(long[] fileEntryIds) throws Exception {...`